### PR TITLE
Updated snapcraft file for local build.

### PR DESCRIPTION
**Pull request by xavi-garcia-mena**
This should be the first functional branch of ubuntu personal store as an independent repository.
I had to update the snapcraft.yaml file in order to build from the current location.

Snapweb is still pulled from its own repository when building.

I've updated the README.md file and I've included a GPLv3 COPYING file as that was the license for snapweb. Not sure if we should change it to LGPL, though.

### CODE REVIEW

**Comment by MarcusTomlinson** [comment]
GPL is standard for applications. LGPL only really makes sense if from this project we were providing a library I'd say.

**Comment by MarcusTomlinson** [comment]
Was holding back on reviewing this until my last PR made it into snapweb. Could you pull the latest from there.

**Comment by xavi-garcia-mena** [comment]
Sure, I'm running some tests under Docker for building with Travis and will pull your last PR

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+     snapcraft
+
+A new ubuntu-personal-store_<version>_<arch>.snap will be created.
+
+# Installing
+
+Once you have a snap built locally, you can test it on your system by doing:
+
+     snap install ubuntu-personal-store_<version>_<arch>.snap --dangerous
```
Comment (MarcusTomlinson) [inline README.md]: We actually need --devmode now. Hoping this will go away when/if a PAM interface is ever made.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The --dangerous flag is necessary for installing locally built snaps, which
+have not been signed by the store.
```
Comment (MarcusTomlinson) [inline README.md]: Need an update to this blurb too

File: snapcraft.yaml.in [hunk 3]
```diff
@@ -1,34 +0,0 @@
-name: ubuntu-personal-store
-version: "0.1"
-summary: A beautiful and functional interface for snap management.
-description: This application allows you to browse and install snap packages on your Ubuntu Personal device.
-
-grade: devel
-confinement: devmode
```
Comment (MarcusTomlinson) [inline snapcraft.yaml.in:7]: Please make sure that the snapcraft,yaml that replaces this one has "confinement: devmode" too

File: snapcraft.yaml.in [hunk 4]
```diff
@@ -1,34 +0,0 @@
...
-      - network
-      - opengl
-      - unity7
-
-parts:
-  ubuntu-personal-store-qml:
-    plugin: dump
-    source: .
-    stage-packages:
-      - gksu:UNKNOWN_ARCH
```
Comment (MarcusTomlinson) [inline snapcraft.yaml.in:23]: The new snapcraft.yaml also needs gksu added to stage-packages

File: authenticate [hunk 5]
```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-ubuntu-personal-store.generate-token
+$SNAP/bin/generate-token
```
Comment (xavi-garcia-mena) [inline authenticate:3]: This seems to fix the issue generating the token in Xenial. 
It looks like it's not happy with calling the command as it is outside the snap package.

File: snapcraft.yaml [hunk 6]
```diff
@@ -27,32 +27,28 @@ apps:
...
       - qml-module-ubuntu-web
     after: [desktop-qt5]
-    organize:
-      ubuntu-personal-store/ubuntu-personal-store.qml: ubuntu-personal-store.qml
-      ubuntu-personal-store/pkg: pkg
-      ubuntu-personal-store/setup: setup
     stage:
       - ubuntu-personal-store.qml
-      - pkg/*
+      - authenticate
```
Comment (xavi-garcia-mena) [inline snapcraft.yaml:40]: I forgot to stage authenticate. 
Anyway, that was not the reason why it was failing in Xenial

**Comment by xavi-garcia-mena** [comment]
@MarcusTomlinson, this should be ready to review again.
It's working again in Xenial. Please, see if the change in the "authenticate" script makes sense to you

**Review by MarcusTomlinson — APPROVED** [review]
:)

**Pull request merged**